README.md

```diff
@@ -24,34 +24,37 @@ npm install -g contentful-export
 Usage: contentful-export [options]
 
 Options:
+  --version             Show version number                            [boolean]
 
---version             Show version number [boolean]
+  --space-id            ID of Space with source data         [string] [required]
 
---space-id            ID of Space with source data [string] [required]
+  --management-token    Management API token for the space to be exported.
+                                                             [string] [required]
 
---rate-limit          How many requests to perform per period of time [number] [default: 6]
+  --export-dir          Defines the path for storing the export json file
+                        (default path is the current directory)         [string]
 
---rate-limit-period   How much time to wait before retry in ms [number] [default: 1000]
+  --include-drafts      Include drafts in the exported entries
+                                                      [boolean] [default: false]
 
---management-token    Management API token for the space to be exported. [string] [required]
+  --skip-roles          Define this flag to skip exporting roles and permissions
```

Comment:
The `Define this flag...` part sounds a bit weird. I would just say `Skip exporting roles and permissions`